src/io.c

```diff
@@ -2165,7 +2171,7 @@ _dispatch_operation_advise(dispatch_operation_t op, size_t chunk_size)
 	}
 	advise.ra_offset = op->advise_offset;
 	op->advise_offset += advise.ra_count;
-#ifdef __linux__
+#if __linux__
```

Comment:
I think this change is wrong, `#if defined()` is what you use elswhere, please revert